Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -138,6 +140,67 @@ public static string GetCacheFileName(Uri uri)
         }
 
         /// <summary>
+        /// Assures that image is available in the cache
+        /// </summary>
+        /// <param name="uri">Uri of the image</param>
+        /// <param name="storeToMemoryCache">Indicates if image should be available also in memory cache</param>
+        /// <returns>void</returns>
+        public static async Task PrecacheAsync(Uri uri, bool storeToMemoryCache = false)
```

Comment:
Not sure if this is the right name. Could also be CacheAheadAsync, AssureCachedAsync or something else.